File: 2177.find-three-consecutive-integers-that-sum-to-a-given-number/solution.erl
-spec sum_of_three(Num :: integer()) -> [integer()].
sum_of_three(Num) ->
  case Num rem 3 == 0 of
    false ->
      [];
    true ->
      [Num div 3 - 1, Num div 3, Num div 3 + 1]
  end.

File: 2177.find-three-consecutive-integers-that-sum-to-a-given-number/solution.js
/**
 * @param {number} num
 * @return {number[]}
 */
var sumOfThree = function(num) {
    if (num % 3 != 0) {
        return [];
    }
    num /= 3;
    return [num-1, num, num+1];
};
